task/conftest-clair/0.1/conftest-clair.yaml

```diff
@@ -33,8 +33,16 @@ spec:
                    timestamp: $date,
                    namespace,
                    successes,
-                   failures: (.failures // [])|map(.metadata.details.name) | unique
+                   failures: (.failures | length)
                  }' $(workspaces.conftest-ws.path)/clair-vulnerabilities.json || true)
+
+        # Log out the failing runs
+        if [ $(echo $HACBS_TEST_OUTPUT | jq '.failures') -gt 0 ]
+        then
+          echo "The clair-vulnerabilities test has failed with the following issues::"
+          echo $(jq '.[].failures // []|map(.metadata.details.name) | unique' $(workspaces.conftest-ws.path)/clair-vulnerabilities.json)
```

Comment:
why echo why not directly execute the `jq` command?

Reply:
Yes, you are right. Let me update it. Thank you!